src/PyCall.jl

```diff
@@ -686,6 +687,33 @@ function pyimport_conda(modulename::AbstractString, condapkg::AbstractString,
     end
 end
 
+"""
+    conda_add(packages::AbstractVector{<: AbstractString})
+
+Install conda `pakcages` _if_ PyCall is configured to use Conda.jl;
+otherwise this function does nothing.  It is intended to be used from
+`deps/build.jl` of the packages depending on PyCall, for safely
+installing Python packages via conda.  More precisely, this function
+detects if Python executable is re-installed during installing
+`packages` and then re-build PyCall if it happens.
+"""
+function conda_add(packages::AbstractVector{<: AbstractString})
+    if !conda
+        @debug "PyCall is not configured to use Conda.jl. Skip installing $packages."
+        return
+    end
+
+    before = Conda.version("python")
+    for pkg in packages
+        Conda.add(pkg)
```

Comment:
TODO: switch to `Conda.add(packages)` once the Conda.jl version containing  https://github.com/JuliaPy/Conda.jl/pull/130 is released.